metrics/influxdb.go

```diff
@@ -0,0 +1,94 @@
+package metrics
+
+import (
+	"fmt"
+	"sync"
+
+	gomet "github.com/rcrowley/go-metrics"
+	"github.com/vrischmann/go-metrics-influxdb"
+)
+
+// Influx :
+type Influx struct {
+	Counter                                map[string]gomet.Counter
+	Histogram                              map[string]gomet.Histogram
+	threadLock, counterLock, histogramLock *sync.Mutex
+	running                                *bool
+}
+
+// SendMetricsNow : implements Metrics interface
+func (influx Influx) SendMetricsNow() {
+	influxdb.SendMetricsNow()
+}
+
+// New :
+func (influx Influx) New() Influx {
```

Comment:
This is not a typical go pattern.
You should have a package function like
func NewInflux(input.....) Influx {
  return Influx{
       .......
   }
}

So other places can just call NewInflux() to create a new influx object, instead of Influx{}.New()